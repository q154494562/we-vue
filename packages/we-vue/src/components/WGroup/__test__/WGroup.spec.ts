import { shallowMount } from '@vue/test-utils'
import Group from '../'

describe('group', () => {
  test('create', () => {
    const wrapper = shallowMount(Group, {
      propsData: {},
    })

    expect(wrapper.contains('.weui-cells')).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('create with title', () => {
    const wrapper = shallowMount(Group, {
      propsData: {
        title: 'test',
      },
    })

    expect(wrapper.contains('.weui-cells__title')).toBeTruthy()
    expect(wrapper.find('.weui-cells__title').text()).toBe('test')

    expect(wrapper.html()).toMatchSnapshot()
  })
})
