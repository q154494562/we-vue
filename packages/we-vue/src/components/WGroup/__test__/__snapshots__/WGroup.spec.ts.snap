// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`group create 1`] = `
<div>
  <div class="weui-cells"></div>
</div>
`;

exports[`group create with title 1`] = `
<div>
  <div class="weui-cells__title">test</div>
  <div class="weui-cells"></div>
</div>
`;
